CardanoSharp.Wallet/AddressService.cs

```diff
@@ -238,6 +240,8 @@ networkType switch
             type switch
             {
                 NetworkType.Testnet => new NetworkInfo(0b0000, 1097911063),
+                NetworkType.Preview => new NetworkInfo(0b0000, 2),
+                NetworkType.Preprod => new NetworkInfo(0b0000, 1),
```

Comment:
TODO: make sure the `NetworkId` is correct

Reply:
I've confirmed that the `NetworkMagic` is correct. I just need to confirm the `NetworkId`

Reply:
According to https://github.com/cardano-foundation/CIPs/blob/master/CIP-0019/README.md#network-tag, the network-tag (aka NetworkId) is `0` for "Testnet(s)".